### DISCUSSION TITLE
No progress bar on Google Colab

### DISCUSSION THREAD

**Discussion by hmtrii** [Lightning-AI/pytorch-lightning#9027] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi everyone, 
I build a model by Pytorch Lightning and train it on Google Colab. However, when I run the training model, the progress bar is not showed and the output seems messy. Could anyone help me out with this problem?
![image](https://user-images.githubusercontent.com/42197959/130319635-3626625a-e076-4c74-993e-cac2df5fca38.png)
This is my code:
```
    tb_logger = pl_loggers.TensorBoardLogger(params['work_dir'])
    callbacks = [
        EarlyStopping(monitor='val_loss', patience=5, check_on_train_epoch_end=False), 
        ModelCheckpoint(monitor='val_loss', save_last=True),
        LearningRateMonitor(logging_interval='step')
    ]
    trainer = pl.Trainer(
        accelerator=params['accelerator'],
        logger=tb_logger, 
        max_epochs=params['epochs'], 
        callbacks=callbacks, 
        check_val_every_n_epoch=3, 
        resume_from_checkpoint=params['resume_from'],
        devices=params['devices']
    )
    trainer.fit(model, datamodule)
```
Thank you ❤️

**Comment by talhaanwarch** [comment]
what is tqdm version? i think you need to upgrade it

**Reply by hmtrii** [reply]
The version is 4.62.0. I upgraded it to 4.62.1, but the problem is still there.

**Reply by talhaanwarch** [reply]
have you restart your kernel after installing? because for me it worked?

**Comment by awaelchli** [comment]
It looks like there is no space for the progress bar. Your screen / window is too small in the horizontal direction. Remove a metric or resize the terminal.
